proposed/http-message.md

```diff
@@ -1161,10 +1161,14 @@ interface UriInterface
      * Implementations SHOULD restrict values to "http", "https", or an empty
      * string but MAY accommodate other schemes if required.
      *
-     * If no scheme is present, this method MUST return an empty string.
+     * The string MUST omit the closing scheme delimiter (":"), if present.
```

Comment:
this should be moved to withScheme to be consistent with withQuery and withFragment.